src/main/java/uk/gov/hmcts/ccd/domain/service/validate/DefaultValidateCaseFieldsOperation.java

```diff
@@ -139,7 +62,13 @@ private boolean hasEventId(CaseTypeDefinition caseTypeDefinition, String eventId
     public void validateData(Map<String, JsonNode> data,
                              CaseTypeDefinition caseTypeDefinition,
                              final CaseDataContent content) {
-        caseTypeService.validateData(data, caseTypeDefinition);
-        validateOrganisationPolicy(caseTypeDefinition.getId(), content);
+        caseTypeService.validateData(
+            createValidationContext(data, caseTypeDefinition)
```

Comment:
we could just use the Validation constructor here maybe

Reply:
DONE